src/Symfony/Component/DependencyInjection/Tests/Compiler/AutowirePassTest.php

```diff
@@ -50,6 +50,22 @@ public function testProcess()
         $this->assertEquals(Foo::class, (string) $container->getDefinition('bar')->getArgument(0));
     }
 
+    /**
+     * @expectedException \Symfony\Component\DependencyInjection\Exception\AutowiringFailedException
+     * @expectedExceptionMessage Cannot autowire service "Symfony\Component\DependencyInjection\Tests\CompilerEslaAction": argument "$notExisting" of method "Symfony\Component\DependencyInjection\Tests\Compiler\ElsaAction::__construct()" has type "Symfony\Component\DependencyInjection\Tests\Compiler\NotExisting" but this class was not found.
+     */
+    public function testProcessNotExisintActionParam()
```

Comment:
This test seems to pass even without your code changes. Is there a way to reproduce easily the original issue?

Reply:
Yes just add an controller or an action, e.g
```services.yml
services:
    _defaults:
        public: true
        autowire: true
        autoconfigure: true

    User\Action\LogOrCreate: ~
```

Reply:
Please add a test that actually covers the fixed bug if this one does not

Reply:
test added @chalasr